R/custom_functions.R

```diff
@@ -603,3 +585,213 @@ prettyDots <-
     print(plot)
     print(plot.noleg)
   }
+
+edgeR_create_fit <- function(object, gene.detection.thres = 0.10, celltype.ident = "order")
```

Comment:
OK for now, but in the future, you might consider making gene.detection.thres a dynamic parameter that can be input when calling the function (rather than hardcoded at 0.10)

Reply:
hmm, I think the current code will let you do this. The gene.detection.thres = 0.10 is the default set when the parameter is left undefined. If explicitly set, it will act as a dynamic parameter. Did I misinterpret this?

Reply:
Ah OK I misunderstood.. That's fine then